android/app/src/main/AndroidManifest.xml

```diff
@@ -19,15 +19,13 @@
   <uses-permission android:name="android.permission.READ_MEDIA_VIDEO" tools:node="remove" />
   <uses-permission android:name="android.permission.READ_MEDIA_VISUAL_USER_SELECTED"
     tools:node="remove" />
-  <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"
-    android:maxSdkVersion="32" />
+  <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
   <!--
     Does nothing in Android 11 (API Level 30) and above; however, `expo-media-library`
     still needs it on Android 11 as it throws an error.
```

Comment:
Probably should update comment to say "...still needs it on Android 11`+` as it throws an error."